courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<ul><li>Remove "as we did in the last notebook" since we didn't do that in the "last notebook" here, right?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Done</p>